### ISSUE TITLE
Payments task - include PayU as an option in India

### ISSUE THREAD

**Issue by pmcpinto** [woocommerce/woocommerce-admin#6172]
**Description:** We need to start including PayU as an option in the Payments task for stores located in India. We aim to include it in the WooCommerce Core 5.2 release (code freeze March 18). (context: pbIJXs-Ab-p2)

The flow should be similar to what we have for other local payments, such as RazorPay in India.

**Acceptance Criteria:**
- [ ] If the user selects India as a store location we should display PayU as an option in the Payments task
- [ ] If the user clicks "Set up" we should automatically install PayU from wp.org (link TBD) 
- [ ] After the plugin is installed we should display the 2nd step where the user can add the PayU credentials. 
- [ ] Mark the step as complete after the user adds the Key details and clicks "proceed".

Logo: Available [here](https://corporate.payu.com/brand-portal/)

Title: PayU for WooCommerce
Description: Enable PayU’s exclusive plugin for WooCommerce to start accepting payments in 100+ payment methods available in India including credit cards, debit cards, UPI, & more!

**Comment by pmcpinto** [comment]
cc @timmyc we need to have this included in the Core March release. I already created a [Cycle 9 release report](https://github.com/woocommerce/woocommerce-admin/issues#workspaces/start--manage-focus-5b0524212133e1068138a6fe/reports/release?release=600eb0fc8c08f84f5198f371) on ZenHub, so we can start adding all the issues that should be included in the cycle.

**Comment by joshuatf** [comment]
>PayU from wp.org (link TBD)

@pmcpinto Do we have this link ready?

>They have an API to validate a merchant’s credentials upon connection. Should we consider using it here?

It looks like their API requires credentials, so I'm not sure whether or not that's something we can easily add here.  Might increase the workload here quite a bit so I'm in favor of skipping this for now.

Estimate: 2 (assuming we don't take on the credential validation, otherwise I'd estimate closer to 8).

**Comment by pmcpinto** [comment]
> @pmcpinto Do we have this link ready?

Not yet. It's pending approval on the .org side

> It looks like their API requires credentials, so I'm not sure whether or not that's something we can easily add here. Might increase the workload here quite a bit so I'm in favor of skipping this for now.


👍 let's keep it simple and don't use their API to validate the merchant's credentials.

**Comment by timmyc** [comment]
As we discussed in the scoping hangout today, the plugin has yet to be published to the wp.org repository, so we will add this to the sprint cycle, but it should be a lower priority issue for now. We could get a PR up with the logic, and then pick up testing out the PR once the PayU plugin lands in the .org repo.

Not adding this to the 2.0 milestone, because it is unlikely to be published by next week.

**Comment by samueljseay** [comment]
@pmcpinto There is still no plugin published on .org as far as I can tell?

**Comment by pmcpinto** [comment]
Correct. They were waiting for approval. Let me see if I can get an update about this.

**Comment by pmcpinto** [comment]
@samueljseay the plugin was approved a few hours ago 😄 Here it goes the link: https://wordpress.org/plugins/payu-india

**Comment by samueljseay** [comment]
@pmcpinto We're now just missing final copy for title, description, credentials step ... Is that everything?

**Comment by pmcpinto** [comment]
Yes. I updated the issue description with the final title and description.
I'm not seeing any key details fields in the PayU settings, so I think we should implement something similar to what we have for Klarna: https://d.pr/i/eEA6e6

The 2nd step would be:
**Title:** Connect to your PayU account
**Description:** PayU can be configured under your store settings [link to PayU settings]. Create your PayU account [here](https://payu.in/).
**CTA:** Continue [redirects the user to the Payments task]

What do you think?

**Comment by samueljseay** [comment]
That sounds good to me. Thanks @pmcpinto 🙇

**Comment by samueljseay** [comment]
@pmcpinto I've used this link "https://onboarding.payu.in/app/account" for the "Create your PayU account" copy because it takes you straight to account creation, is that ok?

**Comment by pmcpinto** [comment]
Makes sense. Thanks for the suggestion!
